src/EnergyPlus/FluidProperties.cc

```diff
@@ -4195,7 +4195,7 @@ namespace Fluid {
 
         // Error handling
         if (error != GlycolError::Invalid && !state.dataGlobal->WarmupFlag) {
-            df->glycolErrorLimits[(int)error] = this->errors[(int)error].count;
+            df->glycolErrorLimits[(int)error] = ++this->errors[(int)error].count;
```

Comment:
Why is df->glycolErrorLimits even needed?

    if (this->errors[(int)error].count <= df->GlycolErrorLimitTest) {

Reply:
That's a fine question, but this definitely cleaned up those files.  I'm going to get this merged but if you want to continue investigating if glycolErrorLimits is really needed, that can happen later.